tests/unit/components/pacbio/PacbioPoolEdit.spec.js

```diff
@@ -245,6 +280,52 @@ describe('pacbioPoolEdit#edit', () => {
       const input = wrapper.find('[data-attribute=insert-size]')
       expect(input.element.value).toEqual('100')
     })
+
+    it('used volume', async () => {
+      expect(wrapper.find('#pool-used-volume').text()).toContain('2')
+      expect(wrapper.find('#tooltip-div').exists()).toBeTruthy()
+    })
+  })
+
+  describe('when inputs are invalid', () => {
+    it('displays an error when itemplate prep kit box barcode is not present', async () => {
+      const input = wrapper.find('[data-attribute=template-prep-kit-box-barcode]')
+      await input.setValue('')
+      await nextTick()
+      expect(
+        wrapper.find('[data-attribute=pool-template-prep-kit-box-barcode-error]').text(),
+      ).toContain('must be present')
+    })
+    it('displays an error when insert size is not present', async () => {
+      const input = wrapper.find('[data-attribute=concentration]')
+      await input.setValue('')
+      await nextTick()
+      expect(wrapper.find('[data-attribute=pool-concentration-error]').text()).toContain(
+        'must be present',
+      )
+    })
+    it('displays an error when insert size is not present', async () => {
+      const input = wrapper.find('[data-attribute=insert-size]')
+      await input.setValue('')
+      await nextTick()
+      expect(wrapper.find('[data-attribute=pool-insert_size-error]').text()).toContain(
+        'must be present',
+      )
+    })
+    it('displays an error when insert size is not present', async () => {
```

Comment:
'displays an error when volume is not present', same name change required for concentration test above

Reply:
Fixed. Copy paste error 🤦‍♀️